docs/src/developers/style.md

```diff
@@ -568,7 +377,69 @@ points more specific to Julia and documentation that uses
    # TODO: Mention also X, Y, and Z.
    ```
    ````
+### Docstrings
+
+- Every exported object needs a docstring
+- All examples in docstrings should be [`jldoctests`](https://juliadocs.github.io/Documenter.jl/stable/man/doctests/)
+- Always use complete English sentences with proper punctuation
+- Do not terminate lists swith punctuation (e.g., as in this doc)
```

Comment:
typo here: `swith`

Reply:
Oops. Too late. Make a new PR?

Reply:
I'll read everything again tomorrow, it's late here, and make a PR then.

Thank you for closing #2104 👍🏼